docs/appendices/release-notes/5.7.1.rst

```diff
@@ -46,4 +46,8 @@ See the :ref:`version_5.7.0` release notes for a full list of changes in the
 Fixes
 =====
 
-None
+- Fixed an issue that caused exceeding the
+  :ref:`cluster.max_shards_per_node <cluster.max_shards_per_node>` when running
+  ``INSERT INTO... VALUES...`` statement with multiple values on partitioned
+  tables.
```

Comment:
```suggestion
- Fixed an issue that allowed ``INSERT INTO`` statements to create more shards
  than allowed by :ref:`cluster.max_shards_per_node
  <cluster.max_shards_per_node>` if the statement resulted in the creation of
  many partitions at once.
```

It's not just `VALUES`, but insert from query as well(?)

Reply:
yes, but insert from query not that visible due to BulkShardCreationLimiter intervention - let's mention it anyway?

Reply:
That mitigates it, but as far as I understand it doesn't completely prevent the issue.

Reply:
yep, limit is still exceeded (slightly) played around locally. I used suggessed ``INSERT INTO`` in the change entry